### DISCUSSION TITLE
No or negligible difference in SPM and DFN plots

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3315] [Q&A] [answered]
When I am trying to run OKane model at a charge rate of 0.3C and discharge 1 C for 1000 cycles including SEI solvent diffusion model, LAM, and Li plating reversible, there is no or little difference between SPM and DFN plots for capacity over cycles.

**Comment by davidhowey** [comment, marked as answer]
This is probably because the SPM is sufficient to capture what is happening at these C-rates and DFN is overkill.
